src/metabase/api/collection.clj

```diff
@@ -79,10 +79,19 @@
   "Valid values for the `?model=` param accepted by endpoints in this namespace."
   #{"card" "collection" "dashboard" "pulse" "snippet"})
 
+; This is basically a union type. defendpoint splits the string if it only gets one
+(def ^:private models-schema (s/conditional #(vector? %) [su/NonBlankString] :else su/NonBlankString))
```

Comment:
Would it be worth doing 

```clj
(def ^:private ModelString
  (apply s/enum valid-model-param-values))

(def ^:private models-schema (s/conditional #(vector? %) [ModelString] :else ModelString))
```

here? So we have that extra validation here
and